src/lib/utils/handleBackButton.js

```diff
@@ -27,6 +27,12 @@ class BackButtonHandler {
   }
 
   handler = action => {
+    const now = Date.now()
+
+    if (now - this.lastPress <= 300) {
+      return BackHandler.exitApp()
```

Comment:
why do we need to exit app if pressed too quickly ? maybe just do nothing ? in this case you don't need to introduce this.lastPress - you could wrap handler onto throttle or debounce

Reply:
have you checked those solutions: 

https://github.com/GoodDollar/GoodDAPP/issues/3302#issuecomment-880574330
for example, the https://www.npmjs.com/package/react-native-minimize package ?

Reply:
I did check those solutions but didn't implement them as the app never crashed on my device, that's why I sent @vldkh to test on his devices